apps/protocol-api/src/sessions.d.ts

```diff
@@ -7,5 +7,6 @@ declare module 'express-session' {
     siwe: SiweResponse;
     nonce?: string;
     linearNonce?: string;
+    discordNonce?: string;
```

Comment:
@alexkeating, what is the purpose of these session variables?

Reply:
This nonce should be passed into the state to prevent crsf attacks